terraform/20-app/cloudwatch.canary.front-end.tf

```diff
@@ -0,0 +1,21 @@
+module "cloudwatch_canary_front_end_screenshots" {
+  source            = "../modules/cloud-watch-canary"
+  create            = true
+  name              = "${local.prefix}-display"
+  s3_access_logs_id = data.aws_s3_bucket.s3_access_logs.id
+  s3_logs_destination = {
+    bucket_id  = module.s3_canary_logs.s3_bucket_id
+    bucket_arn = module.s3_canary_logs.s3_bucket_arn
+  }
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
+
+  schedule_expression = "rate(10 minutes)"
+  timeout_in_seconds  = 600
```

Comment:
What I originally wanted to do is just calculate this from the schedule expression.
But we want this module to be used on demand as well at some point. Like run this canary whenever we deploy